pil-analyzer/src/evaluator.rs

```diff
@@ -213,6 +216,7 @@ impl<'a, T: FieldElement> Value<'a, T> {
         }
     }
 
+    // TODO Maybe we can get rid of this function?
```

Comment:
Added it to https://github.com/powdr-labs/powdr/issues/1106 as well - we can remove it once we turn all type errors into panics, because then I guess the Debug formatter would be fine, or maybe even no formatter at all.